money-monkey-web/app/ui/NewExpenseButton.tsx

```
@@ -0,0 +1,104 @@
+
+  const options = useMemo(() => {
+    console.log(
+      categories
+        .values()
+        .map((x) => x.name)
+        .forEach(console.log)
+    );
+    return (
+      <>
+        <option value="">Select a category:</option>;
+        {categories.values().map((category) => (
```

Biome

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)